storybook/stories/wallets/utils/WalletsWrapper.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { linkTo } from '@storybook/addon-links';
+import startCase from 'lodash/startCase';
+import StoryLayout from '../../_support/StoryLayout';
+import StoryProvider from '../../_support/StoryProvider';
+import StoryDecorator from '../../_support/StoryDecorator';
+import WalletWithNavigation from '../../../../source/renderer/app/components/wallet/layouts/WalletWithNavigation';
+
+export default (story, context) => {
+  const storyWithKnobs = withKnobs(story, context);
+
+  const getItemFromContext = () =>
+    context.story
+      .replace('Wallet UTXO distribution', 'utxo')
+      .toLocaleLowerCase();
+
+  return (
+    <StoryDecorator>
+      <StoryProvider>
+        <StoryLayout activeSidebarCategory="/wallets" storyName={context.story}>
+          {context.story !== 'Empty' && context.story !== 'Wallet Add' ? (
+            <WalletWithNavigation
+              isActiveScreen={item => item === getItemFromContext()}
```

Comment:
This nav can be fixed the same way [I did](https://github.com/input-output-hk/daedalus/pull/1537/commits/63830bf91d8e281ca3eb09e2d2bf35ce97ae3402) for General Settings. It's probably an issue with Stories names vs Components names

Reply:
Thanks Danilo, this helped me figure it out.